src/modelfree/score_agent.py

```diff
@@ -84,9 +89,10 @@ def env_fn(i):
 
     agent_paths = [agent_a_path, agent_b_path]
     agent_types = [agent_a_type, agent_b_type]
-    zipped = zip(agent_types, agent_paths)
+    zipped = list(zip(agent_types, agent_paths))
+
     agents = [load_policy(policy_type, policy_path, venv, env_name, i)
-              for i, (policy_type, policy_path) in enumerate(zipped)]
+              for i, (policy_type, policy_path) in enumerate(zipped[:venv.num_agents])]
```

Comment:
It feels bad to silently ignore some agents. I think if `venv.num_agents == 1` then we should check that `agent_b_type` is equal to some special sentinel value `none`.

Alternatively, we could merge `agent_{a,b}_{path,type}` into `agents_{path,type}` and make these a list of paths and types. Then check that the list is the same length as the number of agents. If `sacred` natively supports lists this seems like the right thing to do. If we have to split things awkardly ourselves then I'm less sure.